charts/kuberpult/run-kind.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+#set -x
+
+# This script assumes that the docker images have already been built.
+# To run/debug/develop this locally, you probably want to run like this:
+# make clean; LOCAL_EXECUTION=true ./run-kind.sh
+
+cd "$(dirname $0)"
+
+#(make -C ../../services/cd-service/)
+
+# prefix every call to "echo" with the name of the script:
+function print() {
+  /bin/echo "$0:" "$@"
+}
+
+cleanup() {
+    print "Cleaning stuff up..."
+    helm uninstall kuberpult-local || print kuberpult was not installed
+    kind delete cluster || print kind cluster was not deleted
+}
+#trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+trap cleanup INT TERM
+cleanup
+
+print 'creating kind cluster with a hostpath to share testdata...'
+kind create cluster --config=- <<EOF || print cluster exists
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraMounts:
+   - hostPath: $(pwd)/../../infrastructure/scripts/create-testdata/
```

Comment:
Why do we need to mount it? The script just talks to the api so it should also work when invoked from outside.

Reply:
we now checkout the repo normally, and the extraMount is gone